packages/ingest/test/test-FtpProviderClient.js

```diff
@@ -14,6 +17,85 @@ const { s3 } = require('@cumulus/aws-client/services');
 const { randomString } = require('@cumulus/common/test-utils');
 const FtpProviderClient = require('../FtpProviderClient');
 
+test.before(async (t) => {
+  process.env.system_bucket = randomString();
+  process.env.stackName = randomString();
+
+  await S3.createBucket(process.env.system_bucket);
+
+  await S3.putFile(
+    process.env.system_bucket,
+    `${process.env.stackName}/crypto/ssh_client_rsa_key`,
+    require.resolve('@cumulus/test-data/keys/ssh_client_rsa_key')
+  );
+
+  await S3.putFile(
+    process.env.system_bucket,
+    `${process.env.stackName}/crypto/private.pem`,
+    require.resolve('@cumulus/test-data/keys/s3_key_pair_provider_private.pem')
+  );
+
+  await S3.putFile(
+    process.env.system_bucket,
+    `${process.env.stackName}/crypto/public.pub`,
+    require.resolve('@cumulus/test-data/keys/s3_key_pair_provider_public.pub')
+  );
+
+  const createKeyResponse = await KMS.createKey();
+  t.context.kmsKeyId = createKeyResponse.KeyMetadata.KeyId;
+});
+
+test.after.always(() => S3.recursivelyDeleteS3Bucket(process.env.system_bucket));
+
+test('FtpProviderClient supports plaintext usernames and passwords', async (t) => {
+  const ftpClient = new FtpProviderClient({
+    host: '127.0.0.1',
+    encrypted: false,
+    username: 'testuser',
+    password: 'testpass',
+    useList: true
+  });
+
+  const files = await ftpClient.list('/');
```

Comment:
Is the FTP client mocked somehow that I'm not seeing? How is the unit tests listing results?

Reply:
It's hitting the FTP server running in Docker.